packages/Python/lldbsuite/test/lang/swift/different_clang_flags/TestSwiftDifferentClangFlags.py

```diff
@@ -21,10 +20,15 @@
 import os.path
 import unittest2
 
+import sys
+if sys.version_info.major == 2:
+    import commands as subprocess
```

Comment:
Isn't there a subprocess in 2.7, too?

Reply:
yes, it just doesn't have the right methods.

Reply:
Should we introduce an lldbutil wrapper?